xnmt/inferences.py

```diff
@@ -32,13 +33,15 @@ class Inference(object):
             * ``score``: output scores, useful for rescoring
     batcher: inference batcher, needed e.g. in connection with ``pad_src_token_to_multiple``
     reporter: a reporter to create reports for each decoded sentence
+    stream: whether to perform streaming inference (one input at a time) from a file
```

Comment:
Is this still needed?

Reply:
No it wasn't, thanks for the catch! Removed.